### PR TITLE
render form elements

### PR DESCRIPTION
**Pull request by geckotang**
- [x] input[type="text"]
- [x] input[type="checkbox"]
- [x] input[type="radio"]
- [x] textarea
- [x] add id & name attribute to form elements

### CODE REVIEW

**Comment by geckotang** [comment]
@watilde I created form components excruding submit button. Please make sure.

File: lib/util/ids.js [hunk 1]
```diff
@@ -0,0 +1,6 @@
+var i  = 0
+module.exports = function (returnVal) {
+  if (returnVal) return 'js-ids-' + i
```
Comment (watilde) [inline lib/util/ids.js]: this option will not be used any more I guess. We can keep it and remove it as well.

**Comment by watilde** [comment]
Let me put +1 on here.

File: lib/util/ids.js (hunk 1 above)
Comment (geckotang) [inline lib/util/ids.js]: @watilde I use it for InputRadio. However, I'm reviewing `<InputRadio/>` and then the option can be removed.

**Comment by watilde** [comment]
then this become LGTM again

**Pull request merged**